### DISCUSSION TITLE
What happens to a tag on a deleted branch?

### DISCUSSION THREAD

**Discussion by acud** [community/community#23918] [New to GitHub]
<p>Hi there!</p>
<p>My question is quite simple: I’d like to tag a commit on a feature branch and push that commit to my repository and then delete the branch. What happens to the tag in that case? I assume that the release in the github page will be untouched, but will the commit be possible to checkout later through the terminal when I checkout the tag directly?</p>
<p>Many thanks!</p>

**Comment by mcaskill** [comment, marked as answer]
<p>The tag and commit would still exist if the branch is deleted. A branch is simply a way to track a collection of commits.</p>
<aside class="onebox stackexchange" data-onebox-src="https://stackoverflow.com/questions/33283350/what-happen-to-git-tags-pointing-to-a-removed-commit/33284908#33284908">
  <header class="source">

      <a href="https://stackoverflow.com/questions/33283350/what-happen-to-git-tags-pointing-to-a-removed-commit/33284908#33284908" target="_blank" rel="noopener nofollow ugc">stackoverflow.com</a>
  </header>

  <article class="onebox-body">
      <a href="https://stackoverflow.com/users/974186/ren%c3%a9-link" target="_blank" rel="noopener nofollow ugc">
    <img alt="Ren&amp;#233; Link" src="https://user-images.githubusercontent.com/29353/181080012-d8f74ab7-9af6-4483-bdb2-72c15ce1fc2c.png" class="thumbnail onebox-avatar" width="128" height="128">
  </a>

<h4>
  <a href="https://stackoverflow.com/questions/33283350/what-happen-to-git-tags-pointing-to-a-removed-commit/33284908#33284908" target="_blank" rel="noopener nofollow ugc">What happen to Git tags pointing to a removed commit</a>
</h4>

<div class="tags">
  <strong>git, git-branch, git-tag</strong>
</div>

<div class="date">
  
  answered by
  <a href="https://stackoverflow.com/users/974186/ren%c3%a9-link" target="_blank" rel="noopener nofollow ugc">
    Ren&amp;#233; Link
  </a>
  on <a href="https://stackoverflow.com/questions/33283350/what-happen-to-git-tags-pointing-to-a-removed-commit/33284908#33284908" target="_blank" rel="noopener nofollow ugc">03:33PM - 22 Oct 15 UTC</a>
</div>

  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
